Gemfile

```diff
@@ -28,6 +28,7 @@ gem 'bootsnap', '>= 1.4.4', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'rack-cors'
```

Comment:
add the version to the gem. You can search for the gem in rubygems.org, then copy the content of the `Gemfile` input field and paste it here in the `Gemfile`. Usually this gem is needed for development and production. So you could place it with the other gems that aren't scoped by a `group :environment`.

https://rubygems.org/gems/rack-cors
`gem 'rack-cors', '~> 1.1', '>= 1.1.1'`